apps/control/src/ui/talent/TalentCard.ui.tsx

```diff
@@ -42,22 +42,24 @@ const TalentCard: FC<TalentCardProps> = ({ data, ...props }) => {
   const onCasterDelete = () => {
     const { [uid]: removedTalent, ...talents } = live.talents
     const saveData: Partial<Live> = { talents }
+    onRemoveActiveCaster()
     setLive(saveData)
     bSave(saveData)
```

Comment:
You are saving twice here.

You are saving in `onRemoveActiveCaster` also you are saving the function itself (`onCasterDelete`)